### DISCUSSION TITLE
Welcome to Poe-Lurker Discussions!

### DISCUSSION THREAD

**Discussion by C1rdec** [C1rdec/PoeLurker#306] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Getrektgasm** [comment]
Have some problems with poe lurker, commands dont paste and typing like human, i mean word by word, What it can be? App run as administrator etc.

**Reply by C1rdec** [reply]
Do you still experience this issue?

**Reply by readysetexplode** [reply]
This also happens to me, instead of things being instant when i click on them, itll open my chat box and type it out (very quickly) and send it. Still works overall, but if i click to move as its typing itll close the chatbox before its finished. Id say about 25% of the time it works normally (instant when i click on invite/trade). This also happens when using the trade website, when i copy the message to send itll sometimes happen instantly, but mostly itll open the chat box and type it all out.

**Reply by C1rdec** [reply]
We figure out that the issue was [Multiplicity](https://www.stardock.com/products/multiplicity/) that was running on @readysetexplode computer. @Getrektgasm Do you use this software?

**Reply by 1125360347** [reply]
Excuse me, could you please provide me with an installation tutorial?

**Comment by 2f2k** [comment]
ive a question. upon starting lurker while poe is on, the mana globe will always have the 'update refresh' symbol and always open a web page to poe lurker. wondering if this is unintended or if there is actually updates to this program every day

**Reply by C1rdec** [reply]
Make sure you dont start Poe Lurker with the setup. You should have a shortcut on your Desktop

**Reply by 2f2k** [reply]
oooh shit ur right thats what ive been doing....thanks man

**Comment by nemrod10** [comment]
Is it possible to move the ui from bottom to top left ?

**Comment by tsufuri** [comment]
Is there anything I have to do to make hotkeys work? I tried setting different hotkeys for trading (like F2, CTRL+1) but they didn't work.

**Comment by rinjuu123** [comment]
Hey guys. Is it normal that incoming trade messages in Korean doesn't get recognized?

**Reply by C1rdec** [reply]
yes #4
